src/test/regress/expected/multi_mx_modifications.out

```diff
@@ -446,6 +446,15 @@ END;
 COMMIT
 -- verify interaction of default values, SERIAL, and RETURNING
 \set QUIET on
+-- make sure this test always returns the same output no matter which tests have run
+SELECT last_value, min_value, max_value FROM app_analytics_events_mx_id_seq \gset
+ALTER SEQUENCE app_analytics_events_mx_id_seq NO MINVALUE NO MAXVALUE;
+SELECT setval('app_analytics_events_mx_id_seq'::regclass, 3940649673949184);
+      setval      
+------------------
+ 3940649673949184
```

Comment:
Nope, this is just the value which was already being used, by doing this I could leave the rest of the test alone and prove that this big change didn't change any behavior.